### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by colinjennings**

### CODE REVIEW

**Comment by colinjennings** [comment]
Sorry for redoing the pull request !

File: README.md [hunk 1]
```diff
@@ -4,110 +4,37 @@ Whom can we blame for using all our disk space?  Find out here.
 
 Developed by Ryan Eckbo, Tashrif Billah, and Isaiah Norton
 
-Table of Contents
```
Comment (tashrifbillah) [inline README.md:7]: Tashrif will provide new table of contents.

File: README.md [hunk 2]
```diff
@@ -4,110 +4,37 @@ Whom can we blame for using all our disk space?  Find out here.
...
+## 2. Install diskusage-logging and required packages
 
-```
-wget https://github.com/tidyverse/lubridate/archive/v1.6.0.tar.gz
-sudo R CMD INSTALL v1.6.0.tar.gz  
-  
-wget https://github.com/Rdatatable/data.table/archive/1.11.4.tar.gz
-sudo R CMD INSTALL 1.11.4.tar.gz
+```bash
+cd ${HOME}
```
Comment (tashrifbillah) [inline README.md]: Remove this line.

File: README.md [hunk 3]
```diff
@@ -4,110 +4,37 @@ Whom can we blame for using all our disk space?  Find out here.
...
+git clone https://github.com/colinjennings/diskusage-logging.git
```
Comment (tashrifbillah) [inline README.md]: Change it to pnlbwh/diskusage-logging.

File: README.md [hunk 4]
```diff
@@ -4,110 +4,37 @@ Whom can we blame for using all our disk space?  Find out here.
...
+cd diskusage-logging
+pip install -r requirements.txt
 ```
 
-Another solution could be to install and activate devtoolset-3-toolchain before installing the above two.
+## 4. Versions
```
Comment (tashrifbillah) [inline README.md]: Remove this header. But the following informative paragraph can stay.

File: README.md [hunk 5]
```diff
@@ -4,110 +4,37 @@ Whom can we blame for using all our disk space?  Find out here.
...
+Current *diskusage-logging* is on the following version of Python:
 
-However, rest of the prerequisites should be directly installated in R:
+    Python 3.6.10 |Anaconda, Inc.| (default, May  8 2020, 02:54:21)
+    [GCC 7.3.0] on linux
 
-  > install.packages(c('knitr', 'ggplot2', 'xtable', 'magrittr', 'bit64'))
+Package versions: (check version info with `pip freeze`)
```
Comment (tashrifbillah) [inline README.md]: Remove this line. Users are expected to know how to check package version.

File: README.md [hunk 6]
```diff
@@ -187,6 +137,7 @@ and place it in `/etc/cron.d/` in `root`'s `cron`:
 `MAILTO` sends all the output of `cron` job to the specified email address. However, `mail` should be 
 set up and functional. 
 
+
```
Comment (tashrifbillah) [inline README.md:117]: Remove this empty line.

**Review by tashrifbillah — COMMENTED** [review]
I shall review `Configuration` to `Mail report` next time.

File: README.md (hunk 2 above)
Comment (tashrifbillah) [reply README.md]: @colinjennings

File: README.md [hunk 7]
```diff
@@ -4,110 +4,36 @@ Whom can we blame for using all our disk space?  Find out here.
...
-    knitr      = 1.22
-    bit64      = 0.9.7
-    lubridate  = 1.6.0
-    data.table = 1.11.5
-    
-
-* NOTE: the current version of `lubridate` and `data.table` in CRAN, requires 
-a newer compiler than is available on CentOS 6. If required, you can install an older version of 
-them directly from source:
+## 1. Create virtual environment (optional)
```
Comment (tashrifbillah) [inline README.md]: Please remove this point and start from ## 2. below. Users are expected to know how to install Python and activate environment.

File: README.md [hunk 8]
```diff
@@ -119,31 +45,54 @@ text file `_config/dirs.txt`, one per line. E.g.
...
 
+## Scripts
     logbigfiles <dir> # finds files > 300M in <dir>, writes csv output to _data/logbigfiles/
+
     logdirsizes <dir> # writes directory sizes in `<dir>` (default depth 3) to `_data/logdirsizes/`
+
     logdirsizesall # calls `logdirsizes` for each directory in _config file `_config/dirs.txt`
+
     mydf # prints file system usage for each directory in _config file `_config/dirs.txt`
+
```
Comment (tashrifbillah) [inline README.md:50]: Is there a reason you included all the white spaces above?

File: README.md [hunk 9]
```diff
@@ -119,31 +45,54 @@ text file `_config/dirs.txt`, one per line. E.g.
...
-# Output
+## Script Output
 
 Each script generates a csv file when you run it, and saves it to its log directory.
-`logdirsizesall` and `logdf` should be run regularly to get a revealing profile of
-disk usage changes.
+
+`logdf` should be run more frequently (~daily) to get a generalized view of total disk usage trends
+
+`logdirsizesall` should be run less frequently (~weekly) to get a more nuanced view of disk usage of individual directories
```
Comment (tashrifbillah) [inline README.md:59]: Thank you for this enhancement with two new headers.

File: README.md [hunk 10]
```diff
@@ -119,31 +45,54 @@ text file `_config/dirs.txt`, one per line. E.g.
...
-which saves the report as `_data/htmlreport/report-*.html`. 
+### To run this in a bash script where python/conda is not installed system-wide (i.e. when using cron):
+
+First, allow executable:
+```bash
+chmod +x html-report-generator/generate_usage_report.py
+```
+Every time you want it to run:
+```bash
+export PATH=/path/to/diskusage/env/bin/:$PATH
```
Comment (tashrifbillah) [inline README.md]: It should be `/path/to/miniconda3/envs/diskusage/bin/`.

File: README.md [hunk 11]
```diff
@@ -119,31 +45,54 @@ text file `_config/dirs.txt`, one per line. E.g.
...
-of last week's disk space changes. Generate it by running:
+The python file `html-report-generator/generate_usage_report.py` generates an HTML report with detailed diskusage data. Run it like this:
+
+```bash
+conda activate /path/to/diskusage/env
+python html-report-generator/generate_usage_report.py
+```
 
-    makereport.sh
+This saves the report as `_data/htmlreport/report-{filesystem_name}-{date}.html`.
```
Comment (tashrifbillah) [inline README.md]: Put this line after line 95.

Comment (tashrifbillah) [reply README.md]: Delete the rest of lines 78-91.

File: README.md (hunk 7 above)
Comment (colinjennings) [reply README.md]: I removed the subsection and integrated it with the title

File: README.md (hunk 2 above)
Comment (colinjennings) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (colinjennings) [reply README.md:50]: Yes, I thought it formatted better on Github and provided more distinction between the scripts. Can revert if wanted.

Comment (tashrifbillah) [reply README.md:50]: No need to revert.

File: README.md (hunk 11 above)
Comment (colinjennings) [reply README.md]: I shortened this entire section, let me know what you think.

File: README.md [hunk 12]
```diff
@@ -5,20 +5,8 @@ Whom can we blame for using all our disk space?  Find out here.
...
-Using [Miniconda](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html), create a new conda environment:
-
-```bash
-conda create --prefix /path/to/diskusage/env python=3.6.10
-conda activate /path/to/diskusage/env
-```
-
-## 2. Install diskusage-logging and required packages
-
+# Install diskusage-logging and Packages
```
Comment (tashrifbillah) [inline README.md]: Bring back the original header.

File: README.md [hunk 13]
```diff
@@ -71,16 +59,7 @@ Each script generates a csv file when you run it, and saves it to its log direct
...
-
-```bash
-conda activate /path/to/diskusage/env
-python html-report-generator/generate_usage_report.py
-```
-
-This saves the report as `_data/htmlreport/report-{filesystem_name}-{date}.html`.
+The python file `html-report-generator/generate_usage_report.py` generates an HTML report with detailed diskusage data.
 
 ### To run this in a bash script where python/conda is not installed system-wide (i.e. when using cron):
```
Comment (tashrifbillah) [inline README.md]: Delete this line.

File: README.md [hunk 14]
```diff
@@ -90,10 +69,12 @@ chmod +x html-report-generator/generate_usage_report.py
 ```
 Every time you want it to run:
 ```bash
-export PATH=/path/to/diskusage/env/bin/:$PATH
+export PATH=/path/to/miniconda3/envs/diskusage/bin/
 html-report-generator/generate_usage_report.py
 ```
 
+This saves the report as `_data/htmlreport/report-{filesystem_name}-{date}.html`.
```
Comment (tashrifbillah) [inline README.md]: Can we have a different string instead of `{filesystem_name}` ?

Comment (tashrifbillah) [reply README.md]: How about `{directory}`?

**Comment by tashrifbillah** [comment]
Merging so Tashrif can have liberty to change.

**Pull request merged**